src/modules/material-requests/queries/find-by-id-material-requests.graphql

```diff
@@ -0,0 +1,10 @@
+query findMaterialRequestsById($id: uuid!) {
+	app_material_requests(where: { id: { _eq: $id } }) {
+		id
+		object_schema_identifier
+		profile_id
+		reason
+		created_at
+		updated_at
```

Comment:
We'll need more than just this info to render the detail view:
![image](https://user-images.githubusercontent.com/1710840/213486282-15910a83-98b5-4a1d-bc7a-4ed3082bc1d8.png)

- maintainerName
- maintainerLogo
- objectName
- objectPid
- requesterName
- requesterEmail
- RequesterUserGroup

Reply:
Adjusted